# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_range_requests.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_range_requests.js
--- a/netwerk/test/unit/test_range_requests.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_range_requests.js	Tue Feb 12 16:08:25 2019 +0000
@@ -129,7 +129,7 @@
 function received_partial_2(request, data) {
   Assert.equal(data, undefined);
   var chan = make_channel("http://localhost:" + port + "/test_2");
-  chan.asyncOpen2(new ChannelListener(received_cleartext, null));
+  chan.asyncOpen(new ChannelListener(received_cleartext, null));
 }
 
 var case_3_request_no = 0;
@@ -157,7 +157,7 @@
 function received_partial_3(request, data) {
   Assert.equal(partial_data_length, data.length);
   var chan = make_channel("http://localhost:" + port + "/test_3");
-  chan.asyncOpen2(new ChannelListener(received_cleartext, null));
+  chan.asyncOpen(new ChannelListener(received_cleartext, null));
 }
 
 var case_4_request_no = 0;
@@ -190,7 +190,7 @@
 // checking length does not work with encoded data
 //  do_check_eq(partial_data_length, data.length);
   var chan = make_channel("http://localhost:" + port + "/test_4");
-  chan.asyncOpen2(new MyListener(received_cleartext));
+  chan.asyncOpen(new MyListener(received_cleartext));
 }
 
 var case_5_request_no = 0;
@@ -218,7 +218,7 @@
   Assert.equal(partial_data_length, data.length);
   var chan = make_channel("http://localhost:" + port + "/test_5");
   chan.setRequestHeader("If-Match", "Some eTag", false);
-  chan.asyncOpen2(new ChannelListener(received_cleartext, null));
+  chan.asyncOpen(new ChannelListener(received_cleartext, null));
 }
 
 var case_6_request_no = 0;
@@ -248,7 +248,7 @@
 // would like to verify that the response does not have Accept-Ranges
   Assert.equal(partial_data_length, data.length);
   var chan = make_channel("http://localhost:" + port + "/test_6");
-  chan.asyncOpen2(new ChannelListener(received_cleartext, null));
+  chan.asyncOpen(new ChannelListener(received_cleartext, null));
 }
 
 const simpleBody = "0123456789";
@@ -296,7 +296,7 @@
   Assert.equal(4, data.length);
   // do it again to get the rest
   var chan = make_channel("http://localhost:" + port + "/test_7");
-  chan.asyncOpen2(new ChannelListener(received_simple, null));
+  chan.asyncOpen(new ChannelListener(received_simple, null));
 }
 
 var case_8_request_no = 0;
@@ -335,7 +335,7 @@
   Assert.equal(4, data.length);
   // do it again to get the rest
   var chan = make_channel("http://localhost:" + port + "/test_8");
-  chan.asyncOpen2(new FailedChannelListener(testFinished, null, CL_EXPECT_LATE_FAILURE));
+  chan.asyncOpen(new FailedChannelListener(testFinished, null, CL_EXPECT_LATE_FAILURE));
 }
 
 var case_9_request_no = 0;
@@ -371,7 +371,7 @@
 function received_partial_9(request, data) {
   Assert.equal(partial_data_length, data.length);
   var chan = make_channel("http://localhost:" + port + "/test_9");
-  chan.asyncOpen2(new ChannelListener(received_simple, null));
+  chan.asyncOpen(new ChannelListener(received_simple, null));
 }
 
 // Simple mechanism to keep track of tests and stop the server
@@ -400,35 +400,35 @@
 
   // Case 2: zero-length partial entry must not trigger range-request
   var chan = make_channel("http://localhost:" + port + "/test_2");
-  chan.asyncOpen2(new Canceler(received_partial_2));
+  chan.asyncOpen(new Canceler(received_partial_2));
 
   // Case 3: no-store response must not trigger range-request
   var chan = make_channel("http://localhost:" + port + "/test_3");
-  chan.asyncOpen2(new MyListener(received_partial_3));
+  chan.asyncOpen(new MyListener(received_partial_3));
 
   // Case 4: response with content-encoding must not trigger range-request
   var chan = make_channel("http://localhost:" + port + "/test_4");
-  chan.asyncOpen2(new MyListener(received_partial_4));
+  chan.asyncOpen(new MyListener(received_partial_4));
 
   // Case 5: conditional request-header set by client
   var chan = make_channel("http://localhost:" + port + "/test_5");
-  chan.asyncOpen2(new MyListener(received_partial_5));
+  chan.asyncOpen(new MyListener(received_partial_5));
 
   // Case 6: response is not resumable (drop the Accept-Ranges header)
   var chan = make_channel("http://localhost:" + port + "/test_6");
-  chan.asyncOpen2(new MyListener(received_partial_6));
+  chan.asyncOpen(new MyListener(received_partial_6));
 
   // Case 7: a basic positive test
   var chan = make_channel("http://localhost:" + port + "/test_7");
-  chan.asyncOpen2(new MyListener(received_partial_7));
+  chan.asyncOpen(new MyListener(received_partial_7));
 
   // Case 8: check that mismatched 206 and 200 sizes throw error
   var chan = make_channel("http://localhost:" + port + "/test_8");
-  chan.asyncOpen2(new MyListener(received_partial_8));
+  chan.asyncOpen(new MyListener(received_partial_8));
 
   // Case 9: check that weak etag is not used for a range request
   var chan = make_channel("http://localhost:" + port + "/test_9");
-  chan.asyncOpen2(new MyListener(received_partial_9));
+  chan.asyncOpen(new MyListener(received_partial_9));
 
   do_test_pending();
 }